README.rst

```diff
@@ -2,166 +2,159 @@
-	below/above the cursor.
+* Readline-like autocomplete.
+	Suggestions displayed as you type.
 
 * Expected parameter list.
 	As in a lot of modern IDEs, bpython will attempt to display a list of
 	parameters for any function you call. The inspect module is tried first,
-	which works with any Python function, and then pydoc if that fails, which
-	seems to be pretty adequate, but obviously in some cases it's simply not
-	possible. I used pyparsing to cure my nested parentheses woes; again, it
-	was nice and easy.
+	which works with any Python function, and then pydoc if that fails.
 
 * Rewind.
-	I didn't call this "Undo" because I thought that would be misleading, but
+	This isn't called "Undo" because it would be misleading, but
 	"Rewind" is probably as bad. The idea is that the code entered is kept in
 	memory and when the Rewind function is called, the last line is popped and
-	the entire code is re-evaluated. As you can imagine, this has a lot of
-	potential problems, but for defining classes and functions, I've found it
-	to be nothing but useful.
+	the entire code is re-evaluated. 
 
 * Pastebin code/write to file.
-	I don't really use the save thing much, but the pastebin thing's great. Hit
-	a key and what you see on the screen will be sent to a pastebin and a URL
-	is returned for you to do what you like with. I've hardcoded
-	paste.pocoo.org in for now, that needs to be fixed so it's configurable.
-	Next release, I promise.
+	Use the <F8> key to upload the screen's contents to pastebin, with a URL returned.
 
 * Flush curses screen to stdout.
-	A featurette, perhaps, but I thought it was worth noting.  I can't
-	personally recall a curses app that does this, perhaps it's often not
-	useful, but when you quit bpython, the screen data will be flushed to
+	When you quit bpython, the screen data will be flushed to
 	stdout, so it basically looks the same as if you had quit the vanilla
 	interpreter.
 
+=============
 Configuration
 =============
 See the sample-config file for a list of available options.  You should save
-your config file as ~/.config/bpython/config (i.e
-$XDG_CONFIG_HOME/bpython/config) or specify at the command line::
+your config file as **~/.config/bpython/config** (i.e
+``$XDG_CONFIG_HOME/bpython/config``) or specify at the command line::
 
 	bpython --config /path/to/bpython/config
 
+============
+Dependencies
+============
+* Pygments
+* requests
+* curtsies >= 0.1.18
+* greenlet
+* Sphinx != 1.1.2 (optional, for the documentation)
+* mock (optional, for the testsuite)
+* babel (optional, for internationalization)
+* watchdog (optional, for monitoring imported modules for changes)
+* jedi (optional, for experimental multiline completion)
+
+If you are using Python 2 before 2.7.7, the following dependency is also
+required:
+
+* `requests[security]`_ 
+
+If you have problems installing cffi which is needed by pyOpenSSL, please take a look at `cffi docs`_.
```

Comment:
I think we should refer to the homepage here instead. It's easier to document all installation issues there than keep this section up-to-date.